### ISSUE TITLE
Lang string crash on server start

### ISSUE THREAD

**Issue by dann1kid** [91loocekaj/vs-wildfarming#5]
```
Game Version: v1.16.3 (Stable)
Loaded Mods: 
14.02.2022 10:04:16: Critical error occurred
System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в Vintagestory.API.Config.TranslationService.Load(Boolean lazyLoad)
   в Vintagestory.API.Config.Lang.Load(ILogger logger, IAssetManager assetManager, String defaultLanguage)
   в Vintagestory.Server.ServerSystemModHandler.OnLoadAssets()
   в Vintagestory.Server.ServerMain.EnterRunPhase(EnumServerRunPhase runPhase)
   в Vintagestory.Server.ServerMain.Launch()
   в ⁭‎​⁭⁪​‭‏⁯‮‪‭‏‮​‬⁮‍⁯‎⁫⁭⁭‭‬⁯​‬‏⁪‬⁫‍⁪‌​⁮‫‍⁭‮.⁬⁯⁪‌⁬⁯​‏‬⁮⁬⁫‌⁬⁮‏‭‏⁭‍‌‏‭‭⁬⁬⁬⁭‮‌‪‮⁮‌⁯⁮‌⁭⁫‮‮()
   в _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )
-------------------------------
```

**Comment by cyraxan** [comment]
confirm, both 1.2.4 and 1.2.5
```
Game Version: v1.16.3 (Stable)
Loaded Mods:
15.02.2022 9:55:30: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Config.TranslationService.Load(Boolean lazyLoad)
   at Vintagestory.API.Config.Lang.Load(ILogger logger, IAssetManager assetManager, String defaultLanguage)
   at Vintagestory.Server.ServerSystemModHandler.OnLoadAssets()
   at Vintagestory.Server.ServerMain.EnterRunPhase(EnumServerRunPhase runPhase)
   at Vintagestory.Server.ServerMain.Launch()
   at ?????????????????????????????????????????.?????????????????????????????????????????()
   at _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )
-------------------------------
```

**Comment by 91loocekaj** [comment]
This has been fixed in 1.2.6. Please put bug reports on the forum page for this mod, I do not actively check Github.
